sigs.yml

```
@@ -115,7 +115,7 @@
     roadmapProjectIDs:
       - 55
   - name: 'Semantic Conventions: K8s'
-    meeting: Every other Wednesday at 08:00 PT
+    meeting: Every other Monday at 07:30 PT
```

```suggestion
    meeting: Every other Tuesday at 07:30 PT
```
I think we all agreed upon Tuesday